### DISCUSSION TITLE
Automation and alerting (such as Storm Watch)

### DISCUSSION THREAD

**Discussion by jaffro2k** [jasonacox/Powerwall-Dashboard#379] [Ideas]
Hi @jasonacox - is there any possibility of retreiving or setting storm watch via the dashboard?  I am having issues with the feature in the app and when I asked Tesla they said they enable storm watch manually!

I know our local weather beaurau have some pretty specific warnings they deliver that are localised (server thunderstorm for example).

I was thinking what if you could bring in weather warnings into the dashboard - and then set charge to 100% if any of them come up locally?

Just a thought :-)

**Comment by jasonacox** [comment]
I like the idea / suggestion. There are two parts:

1) Changing the Powerwall storm watch mode - This may be possible doing something like these tools: https://github.com/jasonacox/pypowerwall/tree/main/tools - I would love to add that if anyone has time to experiment and submit a pull request! :) 

2) Display weather updates on dashboard - I have a national weather service script I created for my area (USA / Los Angeles) that pulls the alerts (see [nws.py](https://github.com/jasonacox/SentryPi/blob/master/nws.py)) and it aligns to what Storm Watch sees. I don't know how to pull the universal alert from Tesla.  If someone figures that out, it could be added as an html panel with some javascript to display in the Dashboard.
